src/main/java/thrift/model/transaction/Expense.java

```diff
@@ -63,6 +65,18 @@ public boolean isSameTransaction(Transaction otherExpense) {
                 && otherExpense.getValue().equals(getValue());
     }
 
+    public boolean getIsJustUpdated() {
+        return isJustUpdated;
+    }
+
+    public void setIsJustUpdated() {
```

Comment:
Is it possible to have just one method `setIsJustUpdated(boolean isJustUpdated)` instead of 2 different methods?

Reply:
Good point